repositories/analytics_repository.go

```diff
@@ -0,0 +1,39 @@
+package repositories
+
+import (
+	"context"
+
+	"github.com/checkmarble/marble-backend/models"
+)
+
+type Metabase interface {
+	GenerateSignedEmbeddingURL(analyticsCustomClaims models.AnalyticsCustomClaims) (string, error)
+}
+
+type MarbleAnalyticsRepository struct {
+	metabase Metabase
+}
+
+func (repo *MarbleAnalyticsRepository) ListAnalytics(ctx context.Context, organizationId string) ([]models.Analytics, error) {
+	var analytics []models.Analytics
+
+	// Add general dashboard
+	generalDashboardUrl, err := repo.metabase.GenerateSignedEmbeddingURL(models.AnalyticsCustomClaims{
+		Resource: map[string]interface{}{
+			"dashboard": 8,
```

Comment:
This should be injected from an env variable, no ? (to separate staging from prod)